src/main/java/com/minecolonies/coremod/entity/ai/mobs/util/MobEventsUtils.java

```diff
@@ -94,9 +96,9 @@ private static void numberOfSpawns(final Colony colony)
 
         final int raidLevel = getColonyRaidLevel(colony);
 
-        numberOfBarbarians = (int) (BARBARIANS_MULTIPLIER * raidLevel);
-        numberOfArchers = (int) (ARCHER_BARBARIANS_MULTIPLIER * raidLevel);
-        numberOfChiefs = (int) (CHIEF_BARBARIANS_MULTIPLIER * raidLevel);
+        numberOfBarbarians = (int) (BARBARIANS_MULTIPLIER * raidLevel/3);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 3 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)